### PR TITLE
Update OTEDetection

### PR DESCRIPTION
**Pull request by druzhkov-paul**

### CODE REVIEW

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

File: models/instance_segmentation/model_templates/coco-instance-segmentation/instance-segmentation-0002/model.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
         type='StandardRoIHead',
         bbox_roi_extractor=dict(
             type='SingleRoIExtractor',
-            roi_layer=dict(type='RoIAlign', out_size=7, sample_num=0),
+            roi_layer=dict(type='RoIAlign', output_size=7, sample_num=0),
```
Comment (DmitriySidnev) [inline models/instance_segmentation/model_templates/coco-instance-segmentation/instance-segmentation-0002/model.py:38]: Here and below should be `sampling_ratio` instead `sample_num`.

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

File: models/instance_segmentation/model_templates/coco-instance-segmentation/instance-segmentation-0002/model.py (hunk 1 above)
Comment (druzhkov-paul) [reply models/instance_segmentation/model_templates/coco-instance-segmentation/instance-segmentation-0002/model.py:38]: Yeah, thanks. I have it in the list, but it's not the main issue so far.

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

**Comment by druzhkov-paul** [comment]
run tests

**Review by Ilya-Krylov — APPROVED** [review]

**Review by DmitriySidnev — APPROVED** [review]

**Comment by druzhkov-paul** [comment]
run tests

**Pull request merged**